src/tests/userLogin.test.js

```diff
@@ -0,0 +1,21 @@
+
+import { use, expect, request } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../app';
+import mockData from './mockData';
+
+use(chaiHttp);
+
+describe('Authentication', () => {
+  it('login with registered email and password', (done) => {
+    request(app).post('/api/v1/auth/login')
+      .send({
+        email: mockData.user2.email,
+        password: mockData.user2.password,
+      }).end((err, res) => {
+        expect(res).to.have.status(404);
+      });
+    done();
+  });
+});
+
```

Comment:
Add tests for login in with invalid credentials

Reply:
I have created separate testing data one for login successfully, other for looking none existing user and another one for invalid credentials.